h2olog

```diff
@@ -883,9 +883,9 @@ quic_type_map =  {
     "quicly__quictrace_recv": ["pn"],
     "quicly__quictrace_recv_ack_delay": ["ack_delay"],
     "quicly__quictrace_lost": ["pn"],
-    "h2o__send_response_header": ["h2o_conn_id", "h2o_req_id", "h2o_header_name", "h2o_header_value"],
-    "h2o__h3_accept": ["h2o_conn_id"],
-    "h2o__h3_close": ["h2o_conn_id"],
+    "h2o__send_response_header": ["h2o_conn", "h2o_req_id", "h2o_header_name", "h2o_header_value"],
+    "h2o__h3_accept": ["h2o_conn"],
+    "h2o__h3_close": ["h2o_conn"],
```

Comment:
I do not think we want to make these changes.

The names have to be consistent to what we have in the .d files (e.g., quicly-probes.d). See #33.